File: Serene3/Serene3.Web/Modules/com/productimage/productimageColumns.cs
using Serenity.ComponentModel;
using System.ComponentModel;

namespace Serene3.com.Columns;

[ColumnsScript("com.productimage")]
[BasedOnRow(typeof(productimageRow), CheckNames = true)]
public class productimageColumns
{
    [EditLink, DisplayName("Db.Shared.RecordId"), AlignRight]
    public int Id { get; set; }
    public string ProductidName { get; set; }
    [EditLink]
    public string Image { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/productimageColumns.ts
import { ColumnsBase, fieldsProxy } from "@serenity-is/corelib";
import { Column } from "@serenity-is/sleekgrid";
import { productimageRow } from "./productimageRow";

export interface productimageColumns {
    Id: Column<productimageRow>;
    ProductidName: Column<productimageRow>;
    Image: Column<productimageRow>;
}

export class productimageColumns extends ColumnsBase<productimageRow> {
    static readonly columnsKey = 'com.productimage';
    static readonly Fields = fieldsProxy<productimageColumns>();
}